docs/users/bpmn-support.adoc

```diff
@@ -30,118 +30,139 @@ the BPMN elements are rendered with their final shapes.
 [cols="1,1,4a", options="header"]
 .Containers
 |===
-|Name
-|Rendering
-|Comments
+.^|Name
```

Comment:
❓ why did you add `.^` and `^.^` before the line? Was there an issue with the existing syntax, or is it an advice from the AsciiDoc documentation?

Reply:
`.^` is for align vertically the text in the box.
`^` is for align horizontally the text in the box.
`^.^` is for align horizontally and vertically the text in the box.

Reply:
`.^` before the headers change nothing, because it is already on 1 row, but to remember and not forget to add it before each row of this table, I prefer to add it 🙂